15_1_Building_the_Regtest.md

```diff
@@ -0,0 +1,24 @@
+# 15.1: Building the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to build a Regtest (Regression Test) by hand to be able to develop new applications without the need to interact with other peers and blocks.
+Bitcoin Core’s regression test mode (regtest mode) lets you instantly create a brand-new private block chain with the same basic rules as testnet—but one major difference: you choose when to create new blocks, so you have complete control over the environment.
+
+## Starting Bitcoind in Regtest Mode
+
+To start your Bitcoind (Bitcoin Daemon) in Regtest mode and create a private Blockchain, you have to use the following command:
+```
+$ bitcoind -regtest -daemon
```

Comment:
I'd like to see some comments on how this interrelates with a normal setup, which users will presumably have from other parts of the tutorial. Should they start from scratch with a newly create VM? Can they run this in parallel with their existing setups? Can they run it parallel, but not simultaneously?